http-server-netty/src/main/java/io/micronaut/http/server/netty/NettyHttpRequest.java

```diff
@@ -627,8 +627,8 @@ public Optional<io.netty.handler.codec.http.HttpRequest> toHttpRequestDirect() {
     }
 
     @Override
-    public @NonNull Optional<ByteBody> byteBodyDirect() {
-        return Optional.of(byteBody());
+    public ByteBody byteBodyDirect() {
```

Comment:
is this now `Nullable`?

Reply:
yes, it's marked as such on the super method